inlanecruising_plugin/include/inlanecruising_plugin/inlanecruising_plugin.h

```diff
@@ -208,6 +208,10 @@ class InLaneCruisingPlugin
    */ 
   Eigen::Isometry2d curvePointInMapTF(const Eigen::Isometry2d& curve_in_map, const lanelet::BasicPoint2d& p, double yaw) const;
 
+  std::vector<double> get_lookahead_speed(const std::vector<lanelet::BasicPoint2d>& points, const std::vector<double>& speeds, const double& lookahead);
```

Comment:
Would be nice to have doc comments for both of these methods, just for consistency.